### ISSUE TITLE
Release failed

### ISSUE THREAD

**Issue by marcelltoth** [stoplightio/json-schema-viewer#102]
## Describe the bug

Last PR merge (#91) failed to release to NPM.

## Additional context

https://app.circleci.com/pipelines/github/stoplightio/json-schema-viewer/727/workflows/daf18561-17e0-4b79-882f-404249d9bde4/jobs/755

![image](https://user-images.githubusercontent.com/543372/103206374-fcc9b200-48fb-11eb-9d2c-5897119f68a9.png)

**Comment by P0lip** [comment]
It seems like the release was correctly performed.
The latest one did in fact fail, but it's because I amended the previous commit (the one that was already released as beta-4 at that time). I forgot to include the breaking change annotation in the message of commit at the time I merged the PR, thus I decided to amend the commit and force push it. There was no code change, hence there is no further action required.
The amended commit wasn't released, but the former one was, so we're good.

![image](https://user-images.githubusercontent.com/9273484/103208862-f3dbdf00-4901-11eb-8bb9-a7b4f037bda0.png)
